src/viam/robot/client.py

```diff
@@ -108,6 +108,18 @@ def with_api_key(cls, api_key: str, api_key_id: str, **kwargs) -> Self:
             """
             Create RobotClient.Options with an API key for credentials and default values for other arguments.
 
+            ::
+
+                # Replace "<API-KEY>" (including brackets) with your machine's API key
+                api_key = '<API-KEY>'
+                # Replace "<API-KEY-ID>" (including brackets) with your machine's API key
+                # ID
```

Comment:
they have  a longer line limit here so we can move these on the same line
```suggestion
                # Replace "<API-KEY-ID>" (including brackets) with your machine's API key ID
```